modules/claims_api/app/workers/claims_api/ews_updater.rb

```diff
@@ -16,14 +16,14 @@ def perform(ews_id)
       ews = ClaimsApi::EvidenceWaiverSubmission.find(ews_id)
       bgs_claim = benefit_claim_service(ews).find_bnft_claim(claim_id: ews.claim_id)
 
-      unless bgs_claim&.dig(:bnft_claim_dto, :filed5103_waiver_ind) == FILE_5103
+      if bgs_claim&.dig(:bnft_claim_dto, :filed5103_waiver_ind) == FILE_5103
+        ews.status = ClaimsApi::EvidenceWaiverSubmission::UPDATED
```

Comment:
we wouldn't want this set to updated until the ind is updated (or was already) and the claim level suspense is completed. both of those have to succeed before we confirm this to be 'updated'